emukit/model_wrappers/gpy_quadrature_wrappers.py

```diff
@@ -143,6 +151,16 @@ def dK_dx1(self, x1: np.ndarray, x2: np.ndarray) -> np.ndarray:
         dK_dx1 = - K[None, ...] * scaled_vector_diff
         return dK_dx1
 
+    def dKdiag_dx(self, x: np.ndarray) -> np.ndarray:
+        """
+        gradient of the diagonal of the kernel (the variance) v(x):=k(x, x) evaluated at x
+
+        :param x: argument of the kernel, shape (n_points M, input_dim)
+        :return: the gradient of the diagonal of the kernel evaluated at x, shape (input_dim, M)
+        """
+        num_points, input_dim = x.shape
+        return np.zeros((input_dim, num_points))
```

Comment:
I guess this is correct while we only have stationary covariance functions implemented but in general I think it should call `self.kernel.dKdiag_dx`?

Reply:
this is the specific wrapper for the RBF kernel which is a stationary kernel.

Reply:
Ah, so it is, I missed that.